app/helpers/reactjs_helper.rb

```diff
@@ -0,0 +1,8 @@
+module ReactjsHelper
+
+  def mount_react_component(name, selector, data)
```

Comment:
Very well, I didn't this of these considerations, just wanted to avoid a repeating pattern of blank div+helper, but if that won't always be the case then disregard my comment.